doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,149 @@
+"""
+=======================
+Background Subtraction using the Rolling Ball Method
+=======================
+
+The rolling ball filter is a segmentation method that aims to separate the
+background from a grayscale image in case of uneven exposure. It is frequently
+used in biomedical image processing and was first proposed by Stanley R.
+Sternberg (1983) in the paper
+[Biomedical Image Processing](https://ieeexplore.ieee.org/document/1654163).
```

Comment:
I'll check out the rendered page, but I guess you need reStructuredText syntax.

Reply:
Sorry, I didn't understand. What is missing from the PR?

Reply:
Sorry I wasn't clear! You need to use reStructuredText syntax, not Markdown. In this case, it would be:
```suggestion
`Biomedical Image Processing <https://ieeexplore.ieee.org/document/1654163>`_.
```
as in, for example: https://github.com/scikit-image/scikit-image/blob/7e4840bd9439d1dfb6beaf549998452c99f97fdd/doc/examples/applications/plot_human_mitosis.py#L42
@emmanuelle Maybe we should document this: Would this place https://scikit-image.org/docs/stable/contribute.html be the right place for it?

Reply:
> Maybe we should document this: Would this place https://scikit-image.org/docs/stable/contribute.html be the right place for it?

I think so! And yes this should be documented! Possibly with a link to the sphinx-gallery docs.

Reply:
> I think so! And yes this should be documented! Possibly with a link to the sphinx-gallery docs.

Done over at #4857!

Reply:
should be fixed now

Reply:
Great. You have used a reference-like syntax (a72d0d79) and I realize it should be a proper reference, indeed. See https://scikit-image.org/docs/dev/auto_examples/segmentation/plot_mask_slic.html to use the same format:
https://github.com/scikit-image/scikit-image/blob/7e4840bd9439d1dfb6beaf549998452c99f97fdd/doc/examples/segmentation/plot_mask_slic.py#L14-L17

Also, you probably want "To implement this algorithm in skimage we can use the following:" to appear afterwards.

Reply:
there is different citation formatting styles in the document you linked as an example. Reference [2] is one style, whereas reference [1] is another (I don't know which ones). Is there a specific format? I'll use Harvard for the time being, and then update the PR to whichever is the desired standard.

Reply:
Oh, I wasn't referring to *citation* formats -- there is currently no standard across the gallery indeed, except in terms of specifying the DOI. Please include the article's DOI exactly as follows:
https://github.com/scikit-image/scikit-image/blob/7e4840bd9439d1dfb6beaf549998452c99f97fdd/doc/examples/segmentation/plot_mask_slic.py#L17

I was referring to reStructuredText syntax (again!). Note that you need a whitespace between `..` and `[1]` (hence, the CI build error). We are aware that the gallery contribution guidelines are lacking, hence this recent PR #4857. As you can read in there, before committing/pushing your changes, you could build the docs locally and check how they render. Please let me know if something is unclear!

Reply:
fixed 2.0